source/includes/v4/_tours.md

```diff
@@ -29,7 +29,7 @@ curl "https://demo.gomus.de/api/v4/tours/categories"
 
 ### Response
 
-The json response contains a list of all tour categories to build up filters. 
+The json response contains a list of all tour categories to build up filters.
```

Comment:
```suggestion
The JSON response contains a list of all tour categories to build up filters.
```

Reply:
Thank you very much for point this out, @huseyin. You are of course very correct - json should be JSON. I have created a new pull request which corrects all of the occurences in the repository where this is the case. See: https://github.com/giantmonkey/gomus-api-doc/pull/30